Makefile

```diff
@@ -31,6 +31,9 @@ deploy-dev: manifests
 	./hack/image_patch_dev.sh
 	kustomize build config/overlays/development | kubectl apply -f -
 
+deploy-test: manifests
```

Comment:
maybe change to `deploy-ci`, `deploy-test` may sound like you can run locally.

Reply:
Updated.